api/v1alpha1/ocivalidator_types.go

```diff
@@ -70,6 +76,16 @@ type Auth struct {
 	SecretName string `json:"secretName" yaml:"secretName"`
 }
 
+type SignatureVerification struct {
+	// Provider specifies the technology used to sign the OCI Artifact
+	// +kubebuilder:validation:Enum=cosign
+	// +kubebuilder:default:=cosign
+	Provider string `json:"provider" yaml:"provider"`
+
+	// SecretName is the name of the Kubernetes Secret that contains trusted public keys used to sign artifacts in the OciRegistryRule
```

Comment:
It would be useful to mention here that the secret referred to by SecretName must exist in the same namespace as the OCIValidator.